include/cloysterhpc/cloyster.h

```diff
@@ -12,16 +12,16 @@
 #include <string>
 
 namespace cloyster {
-bool showVersion = false;
-bool runAsRoot = false;
-bool dryRun = false;
-bool enableTUI = false;
-bool enableCLI = false;
-bool runAsDaemon = false;
-std::string logLevelInput
-    = fmt::format("{}", magic_enum::enum_name(Log::Level::Info));
-std::string answerfile;
-std::string customRepofilePath;
+extern bool showVersion;
+extern bool runAsRoot;
+extern bool dryRun;
+extern bool enableTUI;
+extern bool enableCLI;
+extern bool runAsDaemon;
+
+extern std::string logLevelInput;
+extern std::string answerfile;
+extern std::string customRepofilePath;
```

Comment:
I don't know a better solution to this. This is not a change request and only a comment, but I don't like this either. It's not your code, the old code wasn't good at all also.

Reply:
The old code duplicated the variable instantiations (for some reason, even though we had the include guards)

I moved them to an external file, leaving only the declarations.

I also don't like this, but I could not think about any other way

The other solution would be using a class where every variable is public and static (or a struct where every member is static), but this would take some time to refactor